easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb

```diff
@@ -0,0 +1,36 @@
+# This is an easyconfig file for EasyBuild, see https://easybuilders.github.io/easybuild/
+
+name = 'imkl'
+version = '2018.3.222'
+
+homepage = 'https://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized,
+ extensively threaded math routines for science, engineering, and financial
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'gompi', 'version': '2018b'}
+
+sources = ['l_mkl_%(version)s.tgz']
+checksums = ['108d59c0927e58ce8c314db6c2b48ee331c3798f7102725f425d6884eb6ed241']
+
+dontcreateinstalldir = 'True'
+
```

Comment:
@akesandgren Any reason you dropped the `components = ['intel-mkl']` line here? Cfr. #6234

Reply:
Copied from wrong version.... fixing.